permissions/plugin-lambdatest-automation.yml

```diff
@@ -2,6 +2,6 @@
 name: "lambdatest-automation"
 github: "jenkinsci/lambdatest-automation-plugin"
 paths:
-- "org.jenkins-ci.plugins"
+- "org/jenkins-ci/plugins/lambdatest-automation"
```

Comment:
I _guess_ it's lgtm

* `group = 'org.jenkins-ci.plugins'` (https://github.com/jenkinsci/lambdatest-automation-plugin/blob/master/build.gradle#L16)
* `shortName = 'lambdatest-automation'` (https://github.com/jenkinsci/lambdatest-automation-plugin/blob/master/build.gradle#L28)